src/components/sidebar/routePathView/routePathListTab/RoutePathLinksTab.tsx

```diff
@@ -17,41 +18,88 @@ interface IRoutePathLinksTabProps {
 @inject('routePathStore')
 @observer
 class RoutePathLinksTab extends React.Component<IRoutePathLinksTabProps>{
+    reactionDisposer: IReactionDisposer;
+    listObjectReferences: any;
+    listReference: React.RefObject<HTMLDivElement>;
+
+    constructor(props: IRoutePathLinksTabProps) {
+        super(props);
+        this.listObjectReferences = {};
+        this.listReference = React.createRef<HTMLDivElement>();
+    }
+
     private renderList = (routePathLinks: IRoutePathLink[]) => {
         return routePathLinks.map((routePathLink, index) => {
-            return [
+            this.listObjectReferences[routePathLink.startNode.id] = React.createRef();
+            this.listObjectReferences[routePathLink.id] = React.createRef();
+            const res =  [
                 (
                     <RoutePathListNode
                         key={`${routePathLink.id}-${index}-startNode`}
+                        reference={this.listObjectReferences[routePathLink.startNode.id]}
                         node={routePathLink.startNode}
                         routePathLink={routePathLink}
                     />
                 ), (
                     <RoutePathListLink
                         key={`${routePathLink.id}-${index}-link`}
+                        reference={this.listObjectReferences[routePathLink.id]}
                         routePathLink={routePathLink}
                     />
-                ), index === routePathLinks.length - 1 && (
+                )];
+
+            if (index === routePathLinks.length - 1) {
+                this.listObjectReferences[routePathLink.endNode.id] = React.createRef();
+                res.push(
                     <RoutePathListNode
                         key={`${routePathLink.id}-${index}-endNode`}
+                        reference={this.listObjectReferences[routePathLink.endNode.id]}
                         node={routePathLink.endNode}
                         routePathLink={routePathLink}
-                    />
-                ),
-            ];
+                    />,
+                );
+            }
+            return res;
         });
     }
 
+    private onExtend = () => {
+        const extendedObjects = this.props.routePathStore!.extendedObjects;
+        if (extendedObjects.length === 1) {
+            const id = extendedObjects[0];
+            const item = this.listObjectReferences[id].current;
+            const parentHeight = this.listReference.current!.offsetHeight;
+            this.listReference.current!.scrollTo({
```

Comment:
For me, on chrome, this seems to work as well as
```
item.scrollIntoView({ block:'center', behavior: 'smooth' });
```
The last items on the list seem to not be completely scrolled to on click. Maybe the scrolling happens before the extending?